src/walletSdk/Horizon/Transaction/TransactionBuilder.ts

```diff
@@ -80,39 +87,24 @@ export class TransactionBuilder {
     return this;
   }
 
-  addOperation(op: xdr.Operation): TransactionBuilder {
-    this.builder.addOperation(op);
+  addAssetSupport(asset: IssuedAssetId, trustLimit?: string) {
```

Comment:
could we add return type here?

Reply:
argh I keep forgetting to do these, thanks for the reminder!
https://github.com/stellar/typescript-wallet-sdk/pull/45/commits/df09183d3e27553519ffc953efc023d7068c407d